js/table/tableview.ts

```diff
@@ -281,6 +290,15 @@ export class TableView {
         this.cardsVariantUp = ko.observable<boolean>(false);
         this.cardsVariantDown = ko.observable<boolean>(true);
 
+        this.roundNotification = ko.observable("");
+        this.roundNotificationTimer = ko.observable(0);
```

Comment:
What's the point having `roundNotificationTimer` if essentially you use that as Boolean variable to test is time passed or not, and that value does not used. It is much better to just have `roundNotification` and control visibility of the notification just by checking is it have message or not.